src/main/java/org/systemexception/sparktutorial/pojo/SparkContext.java

```diff
@@ -90,4 +92,19 @@ public void loadJson(String fileName, String outputFolder) {
 		JavaRDD dataRdd = data.toJavaRDD();
 		dataRdd.saveAsTextFile(outputFolder);
 	}
+
+	public int countBlanksAccumulator(String fileName, String outputFolder) {
+		logger.info("Generate accumulator for blank lines in uuid-list.txt");
+		JavaRDD<String> file = sparkContext.textFile(fileName);
+		final Accumulator<Integer> blanksCounter = sparkContext.accumulator(0);
+		JavaRDD<String> fileMap = file.flatMap(
+				(FlatMapFunction<String, String>) line -> {
+					if (line.equals("")) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/leo_4/spark-tutorial/file/2016283064/issues/source?bid=3087676&fileBranchId=3125720#l102)